src/resolvers.js

```diff
@@ -27,6 +27,7 @@ export const resolvers = {
   visitsForMonth: args => visitsForMonth(args.year, args.month),
   yesNo: args => lookupSet('yes_no', args.id),
   yesNos: args => lookupSet('yes_no'),
+  latlng: args => updateHousehold(args.household, args.inPlace),
```

Comment:
you don't need this change 

since you've added the definition to the household type, it will automatically be included in the existing calls to household & households

what this is doing is exposing a NEW query called latlng which we don't need.

you can just delete this line and everything else should work

Reply:
Deleted